variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/VariantClassifier.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.commons.core.models;
+
+/**
+ * Implementation of Variant type classification based on Sequence Ontology (SO) definitions.
+ */
+public class VariantClassifier {
+
+    /***
+     * Get the type of a given variant based on Sequence Ontology (SO) definitions.
+     * See https://docs.google.com/spreadsheets/d/1YH8qDBDu7C6tqULJNCrGw8uBjdW3ZT5OjTkJzGNOZ4E/edit#gid=1433496764
+     * @TODO: might need to be merged eventually into {@link uk.ac.ebi.eva.commons.core.models.AbstractVariant#getType()}
+     *
+     * @param reference Reference allele - expects trimmed allele
+     * @param alternate Alternate allele - expects trimmed allele
+     * @param subSNPClass SubSNP class (as defined in dbSNP variant data)
+     *                    - See https://www.ncbi.nlm.nih.gov/books/NBK21088/table/ch5.ch5_t3/?report=objectonly
+     * @return Type of the variant
+     */
+    public static VariantType getVariantClassification(String reference, String alternate, int subSNPClass) {
+        String alphaRegEx = "[A-Z]+";
+
+        reference = reference.trim();
+        alternate = alternate.trim();
+
+        if (!reference.equals("NOVARIATION")) {
+            if (subSNPClass == 4) {
+                return VariantType.TANDEM_REPEAT;
+            }
+            if (subSNPClass == 5) {
```

Comment:
I can see those two styles in this method:
```
if () {
  return something;
}
if () { 
  return somethingElse;
}
```
and 
```
if () {
  return something;
}
else if () { 
  return somethingElse;
}
```
I don't have a strong preference by any of those, maybe I like more 'else if', but what I would do is just using one of them and not both.